packages/driver/src/cy/multi-domain/unserializable_subject_proxy.ts

```diff
@@ -37,7 +37,7 @@ const failedToSerializeSubject = (type: string) => {
      * @param thisArg this
      * @param argumentsList args passed.
      */
-    apply (target, thisArg, argumentsList) {
+    apply () {
```

Comment:
i wish our linter complained about un-used arguments

Reply:
I agree. We used to use the no-unused-vars rule (or something similar that applies to this situation) but it was removed at some point and I'm not sure why.